pricemonitor/storing/ethereum_nodes.py

```diff
@@ -22,6 +23,8 @@ def etherscan(self, tx):
 class Network(Enum):
     MAINNET = EthereumNetwork(
         nodes=[
+            "https://eth-mainnet.alchemyapi.io/jsonrpc/%s" % os.environ['ALCHEMY_API_KEY'],
```

Comment:
Are these keys mandatory for using the nodes? If so it might be best to check for the env variables earlier when running the script, or perhaps only add the nodes if the env variable is set. The way it is written now, if the environment variables are not set the nodes might be unreachable, delay the script and output errors to the log.